hr_citizenship/security/res_groups.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="res.groups" id="group_multi_citizenship">
+            <field name="name">Show mulcitizenship</field>
```

Comment:
Typo

Reply:
do yo mean noupdate?

Reply:
no. there was a typo mulcitizenship -> multicitizenship. sorry i should be more specific